fiaas_skipper/web/frontend.py

```diff
@@ -39,9 +43,14 @@ def index():
 def status():
     versions = {}
     for tag in ("stable", "latest"):
-        channel = frontend.release_channel_factory("fiaas-deploy-daemon", tag)
-        image = channel.metadata["image"]
-        versions[tag] = image.split(":")[-1]
+        try:
+            channel = frontend.release_channel_factory("fiaas-deploy-daemon", tag)
+            image = channel.metadata["image"]
+            version = image.split(":")[-1]
+        except requests.exceptions.ConnectionError:
```

Comment:
This patch works fine as it is, but I think that the problem could be expressed more clearly by for example catching ConnectionError and logging an error that is useful for troubleshooting the underlying problem (could not connect/bad url) in ReleaseChannel.__call__ and then raise a custom exception with a name that describes the problem that callers (i.e. this method) have to handle (latest version for tag is not available). What do you think?